job_spec/AUTORIFT.yml

```diff
@@ -31,12 +31,18 @@ AUTORIFT:
               minLength: 23
               maxLength: 25
               example: S2A_22WEB_20200627_0_L1C
-            - description: The name of the Landsat 8 or 9 Collection 2 granule to process
+            - description: The name of the Landsat 7, 8 or 9 Collection 2 granule to process
               type: string
-              pattern: "^L[CO]0[89]_"
+              pattern: "^L[ECO]0[789]_"
```

Comment:
As written this allows LC07, LO07, LE08, LE09 which are combinations which don't exist, so I doubt anyone will use them, but I think it's worth being more specific so we remember what our intent was when we look back a few years down the road. We also could add entries to `test_submit_bad_rtc_granule_names` to make it extra clear those aren't valid options.

I think all five platforms can be handled with one pattern of:
```
pattern: "^L([CO]0[89]|E07|[TM]0[45])_"
```
- C/O for 08 and 09
- E for 07
- T/M for 04 and 05

But that's a little dense, and the fact that I felt the need to write out the details in prose suggests trying something more readable. We could do separate entries with separate patterns for each of 89, 7, 45:
```
pattern: "^L[CO]0[89]_"
pattern: "^LE07_"
pattern: "^L[TM]0[45]_"
```
Thoughts? I don't know that this mandates action, and everything else in the PR looks good.